starter-code/index.js

```diff
@@ -1,10 +1,39 @@
-var SortedList = function() {};
+var SortedList = function() {
+    this.items = [];
+    this.length = this.items.length;
+};
 
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+SortedList.prototype.add = function(item) {
+    this.items.push(item);
+    this.length++;    
+    
+    this.items.sort(function(first, second){
+        return first > second;
+    });
+};
 
-module.exports = SortedList;
+SortedList.prototype.get = function(pos) {  
+    return this.items[pos - 1];
+};
+
+SortedList.prototype.max = function() {
+    return this.items[this.length - 1];
+};
+
+SortedList.prototype.min = function() {
+    return this.items[0];
+};
+
+SortedList.prototype.average = function() {
+    return this.items.reduce(function(acc, elem, index, array){
+        return index < array.length -1 ? acc + elem : (acc + elem)/ array.length;
```

Comment:
Por qué dividís entre la longitud en cada iteración en vez de hacerlo al final cuando acaba el reduce?. Igualmente, sería mucho mejor por la reutilización, que hagáis el `reduce()` solo en la función `sum()` y luego llaméis a esta cuando hacéis el `average()`: `return this.sum() / this.items.length`